.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.21'
```

Comment:
Maybe we can also update it to use the `go-version-file` param, in that case we don't need to update it again the next time.

Reply:
Good point!